yt/yt/ytlib/yql_client/proto/yql_service.proto

```diff
@@ -21,6 +21,16 @@ message TYqlQueryFile
 
 ////////////////////////////////////////////////////////////////////////////////
 
+message TYqlCredentials
```

Comment:
Let's maybe call this also secrets? I don't really like two namings "secrets" and "credentials" for the same entities.